webapp/apps/btax/views.py

```diff
@@ -323,10 +322,9 @@ def output_detail(request, pk):
 
     model = url.unique_inputs
     if model.tax_result:
-        output = url.unique_inputs.tax_result
+        tables = url.unique_inputs.tax_result[0]
```

Comment:
hmm.. it looks to me like `tax_result` is always set as the return value of `dropq_get_results`, which is a dictionary, right?  For example, this line:

```
            model.tax_result = dropq_compute.dropq_get_results(normalize(job_ids))
```